Libraries/Components/Pressable/useAndroidRippleForView.js

```diff
@@ -39,25 +46,28 @@ export default function useAndroidRippleForView(
     nativeBackgroundAndroid: NativeBackgroundProp,
   |}>,
 |}> {
+  const {color, borderless, radius} = rippleConfig != null ? rippleConfig : {};
+
   return useMemo(() => {
     if (
       Platform.OS === 'android' &&
       Platform.Version >= 21 &&
-      rippleColor != null
+      (color != null || borderless != null || radius != null)
```

Comment:
What is the expected behavior if color is null, but borderless or radius is specified? I'd imagine nothing should happen, is that what you'd expect?